docs/content/authorization-and-openfga.mdx

```diff
@@ -11,40 +11,34 @@ import { DocumentationNotice, Playground, ProductName, ProductNameFormat } from
 
 ## What Is OpenFGA?
 
-**OpenFGA** is an open source solution to [Fine-Grained Authorization](#what-is-fine-grained-authorization-fga) that applies the concept of [ReBAC](#what-is-relationship-based-access-control-rebac). It was created by the [Okta FGA](https://docs.fga.dev/) team and was inspired by [Zanzibar](#what-is-zanzibar). It was designed for reliability and low latency at a high scale. It offers an HTTP API and has SDKs for programming languages including [Node.js/JavaScript](https://www.npmjs.com/package/@openfga/sdk), [GoLang](https://github.com/openfga/go-sdk), [.NET](https://www.nuget.org/packages/OpenFga.Sdk), [Python](https://pypi.org/project/openfga-sdk), [Java](https://central.sonatype.com/artifact/dev.openfga/openfga-sdk). More SDKs and integrations such as Rego are planned for the future.
+**OpenFGA** is an open source solution to [Fine-Grained Authorization](#what-is-fine-grained-authorization-fga), or **FGA**, that applies the concept of [relationship-based access control](#what-is-relationship-based-access-control-rebac), or **ReBAC**. It was created by the [Okta FGA](https://docs.fga.dev/) team and inspired by [Zanzibar](#what-is-zanzibar). Designed for reliability and low latency at a high scale, FGA offers an HTTP API and SDKs for programming languages like [Node.js/JavaScript](https://www.npmjs.com/package/@openfga/sdk), [GoLang](https://github.com/openfga/go-sdk), [.NET](https://www.nuget.org/packages/OpenFga.Sdk), [Python](https://pypi.org/project/openfga-sdk), and [Java](https://central.sonatype.com/artifact/dev.openfga/openfga-sdk), with additional SDKs and integrations (including Rego) planned for future releases.
 
 <Playground intro={true} />
 
 ## Authentication vs Authorization
 
-[**Authentication**](https://auth0.com/intro-to-iam/what-is-authentication/) (or AuthN) is a process that ensures a user's identity. [**Authorization**](https://en.wikipedia.org/wiki/Authorization) (or AuthZ) means determining if a user can perform a certain action on a particular resource.
+[**Authentication**](https://auth0.com/intro-to-iam/what-is-authentication/) (or **AuthN**) ensures a user's identity. [**Authorization**](https://en.wikipedia.org/wiki/Authorization) (or **AuthZ**) determines if a user can perform a certain action on a particular resource.
 
-For example, when logging in to Twitter, the process in which Twitter verifies the user's username and password are correct is Authentication. The process of checking if that user can view or reply to a certain Tweet is Authorization.
+For example, when you log in to Okta, Authorization verifies that your username and password are correct. Authorization checks if you can access a given Okta tile. [For more information about AuthN vs AuthZ, click here.](https://www.okta.com/identity-101/authentication-vs-authorization/).
```

Comment:
Don't reference Okta for the example please - OpenFGA is a separate entity, we may reference the Okta link for more information about a certain subject, but use a more common neutral brand for the example.

Reply:
Twitter no longer exists and X doesn't really have a strong brand yet. Ideas for a globally-recognizable brand to use? Maybe Facebook?

Reply:
How about Google?

Logging in to Google and checking if you have access to view a Google Drive document?

Reply:
Also minor note - authentication is the process of verification, it does not verify in itself.

An authentication service is the one that verifies, but the authentication describes the process. Same for authorization